src/transformers/models/bridgetower/modeling_bridgetower.py

```diff
@@ -1673,11 +1683,16 @@ def forward(
 
         logits = self.itm_score(pooler_output)
 
+        itm_loss = None
+        if labels is not None:
+            itm_loss = CrossEntropyLoss(logits, labels)
```

Comment:
Here we should instantiate the loss function `loss_fct = CrossEntropyLoss()` as for the MLM loss.

Reply:
Thank @regisss for the suggestion. We have addressed this in the latest commit.